commercetools-models/src/main/java/io/sphere/sdk/orderedits/commands/stagedactions/SetLineItemDistributionChannel.java

```diff
@@ -0,0 +1,37 @@
+package io.sphere.sdk.orderedits.commands.stagedactions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nullable;
+
+public final class SetLineItemDistributionChannel extends OrderEditStagedUpdateActionBase {
+
+    private final String lineItemId;
+    private final Reference<Channel> distributionChannel;
+
+    @JsonCreator
+    private SetLineItemDistributionChannel(final String lineItemId, final Reference<Channel> distributionChannel) {
```

Comment:
use ResourceIdentifiable instead. 

Example for usage can be found here: https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/reviews/commands/updateactions/SetTarget.java

also annotate it as Nullable please